eprihlaska/views.py

```diff
@@ -268,7 +268,8 @@ def admissions_wavers():
 @login_required
 @require_filled_form('admissions_wavers')
 def final():
-    return render_template('final.html', session=session)
+    uid = 9999 + current_user.id
```

Comment:
@JakubNvk Mohlo by sa toto prosim nazvat `specific_symbol` alebo cosi podobne? Ono to uz potom nieje `user_id`, ked sa k tomu prida `9999`.

Vdaka!